package.json

```diff
@@ -253,10 +253,10 @@
     "@babel/traverse": "7.22.5",
     "@types/react": "18.2.74",
     "@types/react-dom": "18.2.23",
-    "react": "19.0.0-beta-04b058868c-20240508",
-    "react-dom": "19.0.0-beta-04b058868c-20240508",
-    "react-is": "19.0.0-beta-04b058868c-20240508",
-    "scheduler": "0.25.0-beta-04b058868c-20240508"
+    "react": "19.0.0-rc-d3ce0d3ea9-20240520",
+    "react-dom": "19.0.0-rc-d3ce0d3ea9-20240520",
+    "react-is": "19.0.0-rc-d3ce0d3ea9-20240520",
+    "scheduler": "0.25.0-rc-d3ce0d3ea9-20240520"
```

Comment:
We should update the `sync-react` script to update `resolutions` as well, i missed this when running it the first time.

Reply:
I'll add support for resolutions. For the other instances I check if we even should specify a version.